swarm/pss/client/client_test.go

```diff
@@ -61,7 +64,8 @@ func TestIncoming(t *testing.T) {
 		t.Fatalf("couldn't make pssmsg: %v", err)
 	}
 
-	pssenv := pss.NewEnvelope(addr, pss.NewTopic(proto.Name, int(proto.Version)), rlpbundle)
+	_ = rlpbundle
```

Comment:
It'll be used in the test when it's fixed. Dont' worry about the client stuff for now please.